src/services/health-check/default-tagline-runner.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Yoast\WP\SEO\Services\Health_Check;
+
+/**
+ * Runs the Default_Tagline health check.
+ */
+class Default_Tagline_Runner implements Runner_Interface {
```

Comment:
```suggestion
class Default_Tagline_Runner implements HealthCheck_Runner_Interface {
```
Ik zou toch graag de interface renamen. Een Runner is wat te vaag. Een HealthCheck_Runner kan maar één ding zijn.

Reply:
@diedexx wat vind jij? omdat het in de Health_Check namespace zit IS het al duidelijk dat het een Health check runner interface moet zijn. Wil jij een besluit nemen?

Reply:
Ik vind het eigenlijk wel nice om 'm Runner_Interface te noemen. Het is minder verleidelijk om een functie die niet `run` heet toe te voegen aan deze interface, dan wanneer de interface `HealthCheck_runner_Interface` heet.